math/pi/spigotpi.go

```diff
@@ -0,0 +1,57 @@
+// spigotpi.go
+// description: A Spigot Algorithm for the Digits of Pi
+// details:
+// implementation of Spigot Algorithm for the Digits of Pi - [Spigot algorithm](https://en.wikipedia.org/wiki/Spigot_algorithm)
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see spigotpi_test.go
+
+package pi
+
+import "strconv"
+
+func PiSpigot(n int) string {
```

Comment:
Could you also change the name of this function - `pi` is already in the package name so `Spigot` should suffice.